### ISSUE TITLE
Dropdown: NVDA failed to read correctly readonly

### ISSUE THREAD

**Issue by arnauddem** [primefaces/primereact#1413]
[x] bug report

Found using primereact 3.4.0, also found in release 4.2.2.
Using various browser: Chrome 83.0.4103.97, Opera 68.0.3618.165, Firefox 77.0.1 
Go to Dropdown widget showcase: https://primefaces.org/primereact/showcase/#/dropdown
choose a city (Paris) from Basic dropdown.

Starts screen reader NVDA (I use version 2020.1), set focus on basic dropdown:
screen reader said: 'edit read only subMenu blank'  (see picture)
![primereact-dropdown](https://user-images.githubusercontent.com/52075241/84889837-d6e46a80-b099-11ea-9549-b0efa3415013.png)

expected result is: 'edit read only subMenu Paris'.

'blank' word comes from <input/> created by renderKeyboardHelper().  I propose to set value of this input, then NVDA will be able to read/talk correct value.   Here is a simple proposal (patch for release 3.4.0, may work only for read only dropdown ... need expert for full solution, thaks).
![primereact-dropdown-2](https://user-images.githubusercontent.com/52075241/84890359-96d1b780-b09a-11ea-8ffb-fb890f744618.png)

**Comment by j-smith-1574** [comment]
Hello, is there any update on the status of this fix?

**Comment by melloware** [comment]
All accessibility issues like this are being addressed in the 9.0 release!

**Comment by AndWillAll** [comment]
Hello, is this being pushed to 10.0?  With government clients, accessibility is a must-have and we need to decide how best to address this issue.  Is there an update on this?  Thanks!

**Comment by melloware** [comment]
Yes it is 10.0 and I think it's being worked on in the background in parallel to the 9.x branch. I could be wrong but that is how previous huge features were implemented!

Trust me PrimeTek is aware how important this is for customers!

**Comment by Doblanko** [comment]
Has this now been pushed to 11.0? Accessibility is critical for our application and we need to decide how to address these issues. Many components are fundamentally not accessible at the moment.

**Comment by melloware** [comment]
Yes been pushed to 11.0.0 as 10.0.0 is focusing on "unstyled mode".

**Comment by habubey** [comment]
Issue tracker is used for defects only as part of our commitment to quality and continuous improvement in all areas. Enhancements are collected as valuable community feedback and managed internally so moving this enhancement ticket to our internal project management backlog.
